### ISSUE TITLE
addContentTypesFromHub not working and 'not supported'

### ISSUE THREAD

**Issue by sparkitect** [SharePoint/sp-dev-docs#8371]
In one customer tenant, the "addContentTypesFromHub" verb does not work; when it is included in a Site Script used by a Site Template, the Site Template is never applied.

Have tried applying the Site Template in the UI and via PnP.PowerShell. Even though a new Microsoft.Online.SharePoint.TenantAdministration.TenantSiteDesignTask is created, it never does _any_ work, regardless of what else is included in the Site Script.

I'm definitely doing it correctly because the same Site Template works in another tenant without issue.

Submitted a Service Request to O365 support and was told
> I would like to inform you that we are from the break-fix team and this issue requires a developer approach. Unfortunately, we do not have the expertise in this issue

and

> we will be proceeding with the closure as this is out of scope of our support boundaries
and that I should seek Developer support instead.

Everything I can find suggests that rollout of the verb is complete accross all tenants, but it doesn't seem so...

Does anyone have any ideas?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 68267476-ee8d-7b95-446b-711ddb683da2
* Version Independent ID: aa099b9d-48db-5513-11c7-32c49a27e5be
* Content: [Site template JSON schema](https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/site-design-json-schema)
* Content Source: [docs/declarative-customization/site-design-json-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/declarative-customization/site-design-json-schema.md)
* Service: **sharepoint-online**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by CallumCrowley** [comment]
@sparkitect have you checked the content type publishing log? (on a site where you have tried to apply the site script)

**Comment by sparkitect** [comment]
Yikes:
![image](https://user-images.githubusercontent.com/9206808/184940584-22039c27-d225-4637-9e8b-32e8a6f31a88.png)

**Comment by CallumCrowley** [comment]
That's actually normal if content types have never been synced to the site (using the old timer job). I think the log gets provisioned the first time the sync gets run on the site. I just assumed that this new method of pushing content types to the site would log any errors incurred in the same place.

**Comment by CallumCrowley** [comment]
Very odd. If the in-library method works correctly and there are no errors with your script, I can't see how Microsoft support wouldn't pick this up for you. This sounds like an issue with the tenant/service.

**Comment by sparkitect** [comment]
definitely, but the O365 support team hasn't a clue about Site Scripts/Templates, apparently:
![image](https://user-images.githubusercontent.com/9206808/184998822-c47eb1f9-af3c-4f49-bfab-3c0c349c50c8.png)

**Comment by CallumCrowley** [comment]
That's bizarre. If you've followed the documentation and done everything correctly, then this is a server side issue. I would definitely go back to Microsoft support, as they are the only people who can deal with server side issues.

**Comment by sparkitect** [comment]
Trust me, you do NOT want to see the length of the email chain w/ them 😠

**Comment by sparkitect** [comment]
I didn't want to have to do this, but I must now call upon the powers that be
and beg for their assistance in my time of great need:

@VesaJuvonen - Master of Masters 🎓
@PopWarner - Speaker of the Air🌊
@hugoabernier - Coder of the Masses 👩‍💻
@patrick-rodgers - Striker of Many Pins 🎳

With your powers combined, etc.
